### PR TITLE
Seeder and factory component, question, answer

### PR DESCRIPTION
**Pull request by dtdat1997**
https://edu-redmine.framgia.vn/issues/6217

### CODE REVIEW

**Comment by dtdat1997** [comment]
Ready

File: database/seeds/QuestionsTableSeeder.php [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+class QuestionsTableSeeder extends Seeder
+{
+    /**
+     * Run the database seeds.
+     *
+     * @return void
+     */
+    public function run()
+    {
+        for ($i=0; $i < 500; $i++) {
```
Comment (BunnyPi04) [inline database/seeds/QuestionsTableSeeder.php]: convention

**Comment by dtdat1997** [comment]
Ready

File: database/factories/QuestionsFactory.php [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+| This directory should contain each of the model factory definitions for
+| your application. Factories provide a convenient way to generate new
+| model instances for testing / seeding your application's database.
+|
+*/
+
+$factory->define(App\Models\Question::class, function (Faker $faker) {
+    return [
+        'component_id' => $faker->numberBetween(1,13),
+        'content_question' => $faker->text.'?',
```
Comment (BunnyPi04) [inline database/factories/QuestionsFactory.php]: convention

**Pull request merged**